cfme/infrastructure/config_management.py

```diff
@@ -170,6 +170,7 @@ class ConfigManagementAllView(ConfigManagementView):
     sidebar = View.nested(ConfigManagementSideBar)
     search = View.nested(Search)
     including_entities = View.include(ConfigManagementEntities, use_parent=True)
+    add_button = Button('Add a Provider')
```

Comment:
I think this should actually be on `ConfigManagementEntities` since its in that section of the page, when the table is not.

Reply:
Makes sense, I will update this

Reply:
Done, also employed a similar strategy for AnsibleTower